stdlib/universe/fill_test.go

```diff
@@ -746,6 +746,658 @@ func TestFill_Process(t *testing.T) {
+					{execute.Time(3), "A"},
+					{execute.Time(4), "B"},
+				},
+			}},
+		},
+		{
+			name: "fill previous empty table",
+			spec: &universe.FillProcedureSpec{
+				DefaultCost: plan.DefaultCost{},
+				Column:      "_value",
+				UsePrevious: true,
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TString},
+				},
+				Data: [][]interface{}{},
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "_value", Type: flux.TString},
+				},
+				Data: [][]interface{}(nil),
+			}},
+		},
+		{
+			name: "null group key",
+			spec: &universe.FillProcedureSpec{
+				Column: "tag0",
+				Value:  values.New(0.0),
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TFloat},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(1), nil, 2.0},
+					{execute.Time(2), nil, nil},
+					{execute.Time(3), nil, 4.0},
+					{execute.Time(4), nil, nil},
+				},
+				GroupKey: execute.NewGroupKey(
+					[]flux.ColMeta{
+						{Label: "tag0", Type: flux.TFloat},
+					},
+					[]values.Value{values.NewNull(semantic.BasicFloat)},
+				),
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TFloat},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(1), 0.0, 2.0},
+					{execute.Time(2), 0.0, nil},
+					{execute.Time(3), 0.0, 4.0},
+					{execute.Time(4), 0.0, nil},
+				},
+				KeyCols:   []string{"tag0"},
+				KeyValues: []interface{}{0.0},
+			}},
+		},
+		{
+			name: "non null group key",
+			spec: &universe.FillProcedureSpec{
+				Column: "tag0",
+				Value:  values.New(0.0),
+			},
+			data: []flux.Table{&executetest.Table{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TFloat},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(1), 1.0, 2.0},
+					{execute.Time(2), 1.0, nil},
+					{execute.Time(3), 1.0, 4.0},
+					{execute.Time(4), 1.0, nil},
+				},
+				GroupKey: execute.NewGroupKey(
+					[]flux.ColMeta{
+						{Label: "tag0", Type: flux.TFloat},
+					},
+					[]values.Value{values.NewFloat(1.0)},
+				),
+			}},
+			want: []*executetest.Table{{
+				ColMeta: []flux.ColMeta{
+					{Label: "_time", Type: flux.TTime},
+					{Label: "tag0", Type: flux.TFloat},
+					{Label: "_value", Type: flux.TFloat},
+				},
+				Data: [][]interface{}{
+					{execute.Time(1), 1.0, 2.0},
+					{execute.Time(2), 1.0, nil},
+					{execute.Time(3), 1.0, 4.0},
+					{execute.Time(4), 1.0, nil},
+				},
+				KeyCols:   []string{"tag0"},
+				KeyValues: []interface{}{1.0},
+			}},
+		},
+	}
```

Comment:
I guess all of these test cases are copies of the test cases from `TestFill_Process`? Instead of copying all these cases, could you just add a new call to `t.Run` in the original test case for the narrow version?

Reply:
This is a temporary change till the narrow version is successfully enabled in the production. Once that is done my plan is to replace the `TestFill_Process_Narrow` with `TestFill_Process` and delete the old test completely.

Reply:
I understand. It's helpful to reviewers if there is less duplicated code, then it would be obvious that the new transformation is just running all the same tests, and they continue to pass. Also, if something goes wrong while we test this out and we need to go back and fix something, it will be less confusing if there is not duplicated code.

There's definitely cases where duplicating code is useful, but I'd rather that it be done sparingly and there seems to be easy ways around it here.

Reply:
That makes sense, I have changed the test to call `t.Run` twice for existing and new cases.